hoot-core/src/main/cpp/hoot/core/ops/MapCropper.cpp

```diff
@@ -398,66 +362,35 @@ void MapCropper::apply(OsmMapPtr& map)
     else
     {
       const Coordinate& c = it->second->toCoordinate();
+      std::shared_ptr<Point> p(GeometryFactory::getDefaultInstance()->createPoint(c));
       LOG_VART(c.toString());
-      if (_envelope.isNull() == false)
```

Comment:
`MapCropper` used to have the option of cropping against an `Envelope` or a `Geometry` and now only uses a `Geometry` for simplicity. So, a lot of this type of code gets cleaned up.